src/libopensc/card-rtecp.c

```diff
@@ -366,6 +366,17 @@ static int rtecp_logout(sc_card_t *card)
 	SC_FUNC_RETURN(card->ctx, SC_LOG_DEBUG_VERBOSE, r);
 }
 
+static int rtecp_set_security_env(	struct sc_card *card,
+									const struct sc_security_env *env,
+									int se_num)
+{
+	assert(env);
+	struct sc_security_env se_env = *env;
+	se_env.flags &= ~SC_SEC_ENV_FILE_REF_PRESENT;
+	assert(iso_ops && iso_ops->set_security_env);
```

Comment:
please avoid `assert()` and use proper error handling

Reply:
fixed args handling. removed assertion here cause didn't find any normal error for that. assertion seemed the best one